jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/ViewResourceDialog.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.explorer
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.ui.layout.panel
+import javax.swing.JComponent
+
+class ViewResourceDialog(project: Project, val resourceType: String, actionTitle: String, val prefix: String) : DialogWrapper(project) {
+    var resourceName = ""
+    private val component by lazy {
+        panel {
+            row("$resourceType:") {
+                textField(::resourceName).constraints(grow).withErrorOnApplyIf("$resourceType must be entered") {
+                    it.text.isNullOrBlank() || it.text.equals(prefix, ignoreCase = true)
+                }
+            }
+        }
+    }
+
+    init {
+        init()
+        title = actionTitle
```

Comment:
i think title needs to be set before init

Reply:
Changed the order, but is there a particular reason for the order, since it works in both cases....